components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/dao/AuthorizationCodeDAOImpl.java

```diff
@@ -204,7 +233,13 @@ public AuthorizationCodeValidationResult validateAuthorizationCode(String consum
             long validityPeriod = 0;
             int tenantId;
 
-            prepStmt = connection.prepareStatement(SQLQueries.VALIDATE_AUTHZ_CODE_WITH_PKCE);
+            String sql;
+            if (OAuth2ServiceComponentHolder.isIDPIdColumnEnabled()) {
+                sql = SQLQueries.VALIDATE_AUTHZ_CODE_WITH_PKCE_IDP_NAME;
```

Comment:
query can be changed as VALIDATE_AUTHZ_CODE_WITH_PKCE_IDP_ID

Reply:
Didn't change the name as within the code it is always the IDP Name that we use... WDYT?